### PR TITLE
#611 RtAlias.email not implemented yet

### LINKED ISSUE #611: RtAlias.email not implemented yet
`com.netbout.client.RtAlias.email()` and `com.netbout.client.RtAlias.email(final String email)` not implemented yet but has to be. Let's implement them and don't forget about the unit test.

### PR DESCRIPTION
**Pull request by mbarbieri**
Fixes #611 
Simply implemented ```RtAlias.email()``` and ```RtAlias.email(final String email)``` methods taking inspiration from other Rt* classes.

### CODE REVIEW

**Comment by karato** [comment]
@mbarbieri Many thanks for the PR, let me find a reviewer for it

**Comment by karato** [comment]
@caarlos0 do this review

File: netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+     * RtAlias can update and retrieve its own mail address.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void updateAndRetrieveEmail() throws Exception {
+        final MkBase base = new MkBase();
+        final User user = this.rule.get();
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final String email = base.randomAlias().email();
+        alias.email(email);
```
Comment (caarlos0) [inline netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java]: @mbarbieri I think you should set another email addr here and assert for it...

**Comment by caarlos0** [comment]
@mbarbieri just one comment above.

File: netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java (hunk 1 above)
Comment (mbarbieri) [inline netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java]: Why?
I'm setting a random mail and assert for it at line 62...
I can set a mail, assert, set another one and assert but I don't see the point if I'm already using a random one..

**Comment by mbarbieri** [comment]
@caarlos0 commented the comment :)

File: netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java (hunk 1 above)
Comment (caarlos0) [inline netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java]: @mbarbieri you are right, sorry, I misunderstood the code the first time I read it.

**Comment by caarlos0** [comment]
@mbarbieri ok

**Comment by caarlos0** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@mbarbieri did you test it on the production server ?
does it return email properly?

**Comment by mbarbieri** [comment]
@dmzaytsev production server doesn't allowed me to change mail address... I'll raise an issue on that. 
I ran tests after starting netbout in local and they all passed

**Comment by dmzaytsev** [comment]
@mbarbieri I talking about `email()` It should return email address,
does it return email properly?

**Comment by mbarbieri** [comment]
@dmzaytsev oh sorry, yes that part was working

**Comment by dmzaytsev** [comment]
@rultor merge please

@mbarbieri thanks!

**Comment by karato** [comment]
@ypshenychka please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor please deploy

**Comment by ypshenychka** [comment]
@caarlos0 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

No issues were found during code review. Try to find at least three problems while future reviews.
Please confirm that everything is clear and you have no questions regarding this approach.

**Comment by ypshenychka** [comment]
@mbarbieri 
According to our [QA Rules](http://www.teamed.io/qa.html):
>Messages in a ticket always start with a name of a user they are addressed to.

Please always indicate a person you're referring too in the beginning of the comment/message.

Please confirm that everything is clear and you have no questions regarding this approach.

**Comment by caarlos0** [comment]
@ypshenychka sure, I'll try

**Comment by vbodak** [comment]
@caarlos0 Thank you

**Comment by ypshenychka** [comment]
@caarlos0 Thank you

**Comment by ypshenychka** [comment]
@mbarbieri Please confirm that you have no questions regarding above remark.

**Comment by mbarbieri** [comment]
@ypshenychka yes it's fine, thanks.

**Comment by ypshenychka** [comment]
@mbarbieri Thanks.

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks for the QA review, we'll work better next time

**Comment by karato** [comment]
@caarlos0 just added 10 mins to @ypshenychka (for QA), payment ID is `58523741`

**19 mins** added to your account (payment number `AP-5XA70035TM115754P`), many thanks for your contribution! 27 hours and 59 mins spent here.

extra minutes for review comments (c=4)

+19 added to your rating, current score is: [+2061](http://www.netbout.com/b/35095?open=rating)

**Pull request merged**